index.html

```diff
@@ -26,28 +26,29 @@ <h1>
          <li><a href="#"><img src="cart.jpg" alt="shopping cart logo">shopping cart (2)</a></li>
       </ul>
       <ul>
-        <img src="shoe2.png" alt="picture of blue and orange sneaker">
-        <img src="shoe3.png" alt="picture of red and black sneaker">
-        <img src="shoe4.png" alt="picture of blue and white sneaker">
+        <img src="shoe2.png" alt="picture of blue and orange sneaker" class="shoe2">
+        <img src="shoe3.png" alt="picture of red and black sneaker" class="shoe3">
+        <img src="shoe4.png" alt="picture of blue and white sneaker" class="shoe4">
     </ul>
     </header>
 
 
-    <img src="shoe1.png" alt="picture of black and white sneaker">
-
+    <img src="shoe1.png" alt="picture of black and white sneaker" class="shoe1">
 
+    <footer>
+      <ul class="left">
+        <li><img class="shoelogo" src="logo.jpg" alt="shoelogo"></li>
+        <li><a href="#">MEN</a></li>
+        <li><a href="#">WOMEN</a></li>
+        <li><a href="#">BOYS</a></li>
+        <li><a href="#">GIRLS</a></li>
+        <li><a href="#">CUSTOMIZE</a></li>
+      </ul>
```

Comment:
This `</ul>` tag was missing, so Chrome thought you had a list within another list! Adding the closing tag makes sure we know that you have two separate lists - one on the left, one on the right.